.github/workflows/ruby_ci.yml

```
@@ -0,0 +1,18 @@
+name: Ruby Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Ruby
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
+    with:
+      plugin: foreman_hooks
```

```suggestion
      plugin: foreman_hooks
      matrix_exclude: '[{"task": "test:foreman_hooks"}]'
```

this should exclude the failing (non-existant) test and allow us to use the action for at least verifying the app boots with that engine.

---

and it works